index.html

```diff
@@ -34,7 +34,9 @@
     
     <div id='side'>
     	
-    	<a href='#' class='equation-tile add' id='add'>Add Equation</a>
+    	<a href='#' class='equation-tile add' id='add-implicit'>Add Implicit Equation</a>
```

Comment:
Maybe leave this as "Add Equation" -- people use this to plot y = f(x) style equations without knowing what implicit equations are.